kola/tests/misc/files.go

```diff
@@ -94,9 +94,19 @@ func sugidFiles(m platform.Machine, validfiles []string, mode string) error {
 func DeadLinks(c cluster.TestCluster) error {
 	m := c.Machines()[0]
 
-	output, err := m.SSH(`sudo find / /usr -ignore_readdir_race -xdev -xtype l`)
+	ignore := []string{
+		"/dev",
+		"/proc",
+		"/run/udev/watch",
+		"/sys",
+		"/var/lib/rkt",
```

Comment:
Okay, I've added Docker to the list.

It was using `-xdev` to avoid the bad links in `/proc` and `/run`, but I removed it in favor of excluding the paths.